schema/vrs.yaml

```diff
@@ -410,52 +472,56 @@ definitions:
         $ref: "#/definitions/CURIE"
 
   # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
-  # Quantity
+  # Numerics, Comparators, and Ranges
 
-  Quantity:
+  Numeric:
```

Comment:
I would prefer that we standardized on nouns (or adjectival forms) as type names as we've done in the remainder of the spec. i.e., Number rather than Numeric, and SimpleNumber rather than SimpleNumeric.

Reply:
agreed. I think ComparatorNumber is odd, so we may want to reconsider our options on that name, Ray?  https://www.mathstips.com/point-ray-line-and-line-segment/
So for our sequencelocations we have an interval who's endpoints are Points (SimpleNumber), Rays (ComparatorNumber) and Segments (Range). These non-geometric names work well if we only needed these datatypes for our linear sequences, but we need them to work for amounts as well, so I suppose ComparatorNumber is clearly something that folks will have to learn about before applying - which is probably a good thing.

Reply:
Sounds fine to me. As these are intended to be general-purpose (not just coordinates), I think that leaves us with:

- Number
  - SimpleNumber
  - ComparatorNumber

Reply:
Agreed: use Number rather than Numeric.